typescript/src/schema/proto.ts

```diff
@@ -59,6 +59,13 @@ interface MethodDescriptorProto
   // into x-goog-request-params header, the array will contain
   // [ ['request', 'foo'], ['request', 'bar']]
   headerRequestParams: string[][];
+  // if 1; send implicit header request. If 2, send dynamic header request. If 3, do not send a header at all.
+  headerRequest: number;
```

Comment:
For better readability, could we use boolean to indicate the dynamic header. See below for sample refactor code.